### DISCUSSION TITLE
Task dependency

### DISCUSSION THREAD

**Discussion by Tandoy** [apache/dolphinscheduler#6263] [Ideas]
this discussion is from issue #6039

**Describe**
Now after selecting a task to prohibit execution in the workflow, the downstream tasks that depend on it will still be executed.
Sometimes it is necessary to prohibit the execution of the current task, and subsequent tasks will not be executed. This will also help shorten the complement time.

**Describe the solution you'd like**
Maybe we can add an option for prohibiting execution and its downstream tasks next to the prohibiting execution button

**Additional context**
![130894326-30ca9a70-c4f4-49bc-be26-412562352380 (1)](https://user-images.githubusercontent.com/56899730/133870761-de8ce4f4-8acc-41b9-a9e9-45c193860901.png)

**Comment by davidzollo** [comment]
good idea
is there anyone who want to implement this feature?
